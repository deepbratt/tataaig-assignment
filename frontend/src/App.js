import "bootstrap/dist/css/bootstrap.min.css";
import React from "react";
import "./App.css";
import Main from "./screens/home";
function App() {
  return (
    <div className="App">
      <Main />
    </div>
  );
}

export default App;
