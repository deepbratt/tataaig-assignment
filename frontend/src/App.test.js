import React from "react";
import { Provider } from "react-redux";
import renderer from "react-test-renderer";
import { PersistGate } from "redux-persist/integration/react";
import App from "./App";
import persist from "./store";
jest.mock("react-router-dom", () => ({
  ...jest.requireActual("react-router-dom"),
  useParams: jest
    .fn()
    .mockReturnValue({ environment: "dev", service: "fakeService" }),
}));

jest.mock("react-redux", () => ({
  ...jest.requireActual("react-redux"),
  useSelector: jest.fn(),
}));
const persistStore = persist();
test("App component snapshot", () => {
  const component = renderer.create(
    <Provider store={persistStore.store}>
      <PersistGate persistor={persistStore.persistor}>
        <App />
      </PersistGate>
    </Provider>
  );
  expect(component.toJSON()).toMatchSnapshot();
});
