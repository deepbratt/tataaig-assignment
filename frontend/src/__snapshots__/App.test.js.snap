// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component snapshot 1`] = `
<div
  className="App"
>
  <div
    className="App"
  >
    <div
      style={
        Object {
          "alignContent": "center",
          "flex": 1,
          "justifyContent": "center",
        }
      }
    >
      <div
        className="fade alert alert-primary show"
        role="alert"
      >
        Welcome to Diet Checker
      </div>
    </div>
    <div
      className="container"
      padder={10}
      style={
        Object {
          "marginBottom": 10,
        }
      }
    >
      <div
        className="row"
      >
        <div
          className="col-md-6 col-sm-12"
        >
          <div
            className="container"
            padder={10}
          >
            <div
              className="card"
            >
              <div
                className="card-header"
              >
                Login
              </div>
              <div
                className="card-body"
              >
                <form
                  className=""
                  onSubmitCapture={[Function]}
                >
                  <div
                    className="form-group"
                  >
                    <input
                      className="form-control"
                      defaultValue={null}
                      id="formBasicEmail"
                      name="email"
                      onChange={[Function]}
                      placeholder="Enter email"
                      required={true}
                      type="email"
                    />
                  </div>
                  <div
                    className="form-group"
                  >
                    <input
                      className="form-control"
                      defaultValue={null}
                      id="formBasicPassword"
                      name="password"
                      onChange={[Function]}
                      placeholder="Password"
                      required={true}
                      type="password"
                    />
                  </div>
                  <button
                    className="btn btn-primary"
                    disabled={false}
                    type="submit"
                  >
                    Login
                  </button>
                </form>
              </div>
            </div>
          </div>
        </div>
        <div
          className="col-md-6 col-sm-12"
        >
          <div
            className="container"
            padder={10}
          >
            <div
              className="card"
            >
              <div
                className="card-header"
              >
                Signup
              </div>
              <div
                className="card-body"
              >
                <form
                  className=""
                  onSubmitCapture={[Function]}
                >
                  <div
                    className="form-group"
                  >
                    <input
                      className="form-control"
                      defaultValue={null}
                      id="firstName"
                      name="firstName"
                      onChange={[Function]}
                      placeholder="Enter first name"
                      required={true}
                      type="text"
                    />
                  </div>
                  <div
                    className="form-group"
                  >
                    <input
                      className="form-control"
                      defaultValue={null}
                      id="lastName"
                      name="lastName"
                      onChange={[Function]}
                      placeholder="Enter last name"
                      required={true}
                      type="text"
                    />
                  </div>
                  <div
                    className="form-group"
                  >
                    <input
                      className="form-control"
                      defaultValue={null}
                      id="semail"
                      name="email"
                      onChange={[Function]}
                      placeholder="Enter email"
                      required={true}
                      type="email"
                    />
                  </div>
                  <div
                    className="form-group"
                  >
                    <input
                      className="form-control"
                      defaultValue={null}
                      id="spassword"
                      name="password"
                      onChange={[Function]}
                      placeholder="Password"
                      required={true}
                      type="password"
                    />
                  </div>
                  <button
                    className="btn btn-primary"
                    disabled={false}
                    type="submit"
                  >
                    Signup
                  </button>
                </form>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
